bodo/pandas/physical/aggregate.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include <memory>
+#include <utility>
+#include "../libs/_array_utils.h"
+#include "expression.h"
+#include "operator.h"
+
+/**
+ * @brief Physical node for count_star().
+ *
+ */
+class PhysicalCountStar : public PhysicalSource, public PhysicalSink {
+   public:
+    explicit PhysicalCountStar() : local_count(0), global_count(0) {}
+
+    virtual ~PhysicalCountStar() = default;
+
+    void Finalize() override {
+        int result =
+            MPI_Allreduce(&local_count, &global_count, 1,
+                          MPI_UNSIGNED_LONG_LONG, MPI_SUM, MPI_COMM_WORLD);
+        if (result != MPI_SUCCESS) {
+            throw std::runtime_error(
+                "PhysicalCountStar::Finalize MPI_Allreduce failed.");
+        }
+    }
+
+    OperatorResult ConsumeBatch(std::shared_ptr<table_info> input_batch,
+                                OperatorResult prev_op_result) override {
+        local_count += input_batch->nrows();
+        return prev_op_result == OperatorResult::FINISHED
+                   ? OperatorResult::FINISHED
+                   : OperatorResult::NEED_MORE_INPUT;
+    }
+
+    std::shared_ptr<table_info> GetResult() override {
+        throw std::runtime_error(
+            "GetResult called on a PhysicalCountStar node.");
+    }
+
+    const std::shared_ptr<bodo::Schema> getOutputSchema() override {
+        std::vector<std::unique_ptr<bodo::DataType>> types;
+        types.emplace_back(std::make_unique<bodo::DataType>(
+            bodo_array_type::arr_type_enum::NULLABLE_INT_BOOL,
+            Bodo_CTypes::CTypeEnum::UINT64));
+        std::vector<std::string> names = {std::string("count_star()")};
+        return std::make_shared<bodo::Schema>(std::move(types), names);
```

Comment:
Add trivial metadata field also for consistency?

Reply:
think I did it but please verify.

Reply:
I don't see changes here.